src/lib/php/common-sysconfig.php

```diff
@@ -195,56 +195,51 @@ function Populate_sysconfig()
 {
   global $PG_CONN;
 
-  $Columns = "variablename, conf_value, ui_label, vartype, group_name, group_order, description, validation_function, option_value";
+  $Columns = array("variablename", "conf_value", "ui_label", "vartype", "group_name",
```

Comment:
change the variable name according to https://github.com/fossology/fossology/wiki/Coding-Style#coding-guidelines-for-php